source/Calamari.Tests/Fixtures/StructuredVariables/Approved/JsonFormatVariableReplacerFixture.ShouldFallBackToStringIfTypePreservationFails.approved.json

```diff
@@ -0,0 +1,9 @@
+{
+  "null2num": "33",
+  "null2obj": "{\"x\": 1}",

```

Comment:
Is this the same behaviour as the JSON replacer?

Reply:
This _is_ testing the JSON replacer. I wanted to explicitly confirm the existing behaviour.

Reply:
🤦